src/rpc/client.cpp

```diff
@@ -208,7 +209,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "createwallet", 1, "disable_private_keys"},
     { "createwallet", 2, "blank"},
     { "createwallet", 4, "avoid_reuse"},
-    { "createwallet", 5, "load_on_startup"},
+    { "createwallet", 5, "descriptors"},
+    { "createwallet", 6, "load_on_startup"},
```

Comment:
> 16528: Note: this is a breaking change for createwallet rpc

should we swap no5 and no6 such as
```
    { "createwallet", 5, "load_on_startup"},
    { "createwallet", 6, "descriptors"},
```
to prevent breaking changes?

If there's any other breaking changes then no need workarounds like that.

Reply:
re-ordered: https://github.com/dashpay/dash/pull/5579/commits/c243ea9096dbb865f41d3dee18950185134439cd